apps/dh/api-dh/source/DataHub.WebApi/Modules/MarketParticipant/User/UserOverviewItemNode.cs

```diff
@@ -50,7 +53,7 @@ public static async Task<CollectionSegment<UserOverviewItemDto>> GetUsersAsync(
             pageNumber,
             pageSize,
             sortProperty,
-            sortDirection,
+            sortDirection.FromNullableSortingToMarketParticipantSorting(),
```

Comment:
suggestion: using the same `.Value` trick mentioned earlier, change this to use `FromSortingToMarketParticipantSorting` instead.